addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -842,6 +842,22 @@ var FieldX2Many = AbstractField.extend({
              }).column_invisible;
         });
     },
+    /**
+     * Computes the default renderer to use depending on the view type.
+     * We create this as a method so we can override it if we want to use
+     * another renderer instead (eg. section-and-note).
```

Comment:
rename section-and-note to section_and_note_one2many (the new name of the widget)